refact_webgui/webgui/selfhost_fastapi_completions.py

```diff
@@ -252,6 +252,10 @@ def __init__(self,
         self._model_assigner = model_assigner
         self._timeout = timeout
 
+    @property
+    def _caps_version(self) -> int:
```

Comment:
looks like we already have changes about _caps_version in dev

Reply:
plus I guess you better merge this changes into oleg's branch instead of dev due to upcoming release